### PR TITLE
pkg: pod added WithAdditionalInitContainer func

### PR DESCRIPTION
**Pull request by kononovn**

### CODE REVIEW

File: pkg/pod/pod.go [hunk 1]
```diff
@@ -701,6 +701,28 @@ func (builder *Builder) WithAdditionalContainer(container *v1.Container) *Builde
...
+// WithAdditionalInitContainer appends additional init container to pod.
+func (builder *Builder) WithAdditionalInitContainer(container *v1.Container) *Builder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Adding new container %v to pod %s", container, builder.Definition.Name)
+	builder.isMutationAllowed("additional container")
+
+	if container == nil {
```
Comment (achuzhoy) [inline pkg/pod/pod.go:713]: do we want a glog entry?

File: pkg/pod/pod.go [hunk 2]
```diff
@@ -701,6 +701,28 @@ func (builder *Builder) WithAdditionalContainer(container *v1.Container) *Builde
...
+		builder.errorMsg = "'container' parameter cannot be empty"
+	}
+
+	if builder.errorMsg != "" {
```
[the remark this answers is not in the document]
Comment (kononovn) [reply pkg/pod/pod.go:717]: no need for glog here.

File: pkg/pod/pod.go [hunk 3]
```diff
@@ -701,6 +701,30 @@ func (builder *Builder) WithAdditionalContainer(container *v1.Container) *Builde
 	return builder
 }
 
+// WithAdditionalInitContainer appends additional init container to pod.
+func (builder *Builder) WithAdditionalInitContainer(container *v1.Container) *Builder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Adding new container %v to pod %s", container, builder.Definition.Name)
```
Comment (trewest) [inline pkg/pod/pod.go:710]: Please add namespace to glog

**Review by trewest — CHANGES_REQUESTED** [review]
Just one small nit

**Review by trewest — APPROVED** [review]
/lgtm

**Pull request merged**